pkg/tasks/repository_snapshot.go

```diff
@@ -257,18 +262,33 @@ func (sr *SnapshotRepository) findOrCreatePulpRepo(repoConfigUUID string, remote
 	return *repoResp.PulpHref, nil
 }
 
+func urlIsRedHat(url string) bool {
+	return strings.Contains(url, "cdn.redhat.com")
+}
+
 func (sr *SnapshotRepository) findOrCreateRemote(repoConfig api.RepositoryResponse) (string, error) {
+	var clientCertPair *string
+	var caCert *string
+	if repoConfig.OrgID == config.RedHatOrg && urlIsRedHat(repoConfig.URL) {
+		clientCertPair = config.Get().Certs.CdnCertPairString
+		ca, err := external_repos.LoadCA()
+		if err != nil {
+			log.Err(err).Msg("Cannot load red hat ca file")
+		}
+		caCert = pointy.Pointer(string(ca))
+	}
+
 	remoteResp, err := sr.pulpClient.GetRpmRemoteByName(repoConfig.UUID)
 	if err != nil {
 		return "", err
 	}
 	if remoteResp == nil {
-		remoteResp, err = sr.pulpClient.CreateRpmRemote(repoConfig.UUID, repoConfig.URL)
+		remoteResp, err = sr.pulpClient.CreateRpmRemote(repoConfig.UUID, repoConfig.URL, clientCertPair, clientCertPair, caCert)
```

Comment:
I tested syncing and it's working, so I think there's no issue here. Just a question. You pass `clientCertPair` for both `clientCert` and `clientKey`. Is the "pair" a combination of the two? And pulp can parse it either way?

Reply:
yep, the pair is both combined.  Pulp (and most http clients) will pull out the right one.  This is how we're doing it with introspection so i just kept it the same.